.github/workflows/continous-integration.yml

```diff
@@ -145,9 +145,10 @@ jobs:
       USE_S3: False
       X_FRAME_OPTIONS: DENY
       XSS_PROTECTION: True
-      CSP_FONT_SRC: "'self' https://fonts.gstatic.com https://fonts.googleapis.com https://code.cdn.mozilla.net"
-      CSP_SCRIPT_SRC: "'self' 'unsafe-inline' https://www.google-analytics.com/analytics.js http://*.shpg.org/ https://comments.mozillafoundation.org/ https://airtable.com https://platform.twitter.com https://cdnjs.cloudflare.com/ajax/libs/gsap/3.8.0/gsap.min.js https://cdnjs.cloudflare.com/ajax/libs/gsap/3.8.0/ScrollTrigger.min.js"
-      CSP_STYLE_SRC: "'self' 'unsafe-inline' https://code.cdn.mozilla.net https://fonts.googleapis.com https://platform.twitter.com"
+      CSP_CONNECT_SRC: "*"
```

Comment:
This one's new. Did the FA script need `connect-src` in addition to everything else or is this from the integration tests? Would it be possible to narrow this down from `*`?

Reply:
Hi @mtdenton 

> Would it be possible to narrow this down from `*`?

I think so. But looking at what we have on production, we have `*` set so that's why I changed it to follow that.

Reply:
Cool, sounds like we just need to take a good pass at our CSP headers anyhow.

Reply:
Agreed 😅